src/compiler/diagnosticMessages.json

```diff
@@ -3059,6 +3059,10 @@
         "category": "Error",
         "code": 2796
     },
+    "Expression requires comparison of excessively large unions or intersections.": {

```

Comment:
Change to "...excessively large union or intersection types."

Reply:
Will do.